config/routes.rb

```diff
@@ -7,7 +7,7 @@
   # Can be used by load balancers and uptime monitors to verify that the app is live.
   get "up" => "rails/health#show", as: :rails_health_check
 
-  resources :posts, only: :index, param: :slug
+  resources :posts, except: %i[new edit destroy update], param: :slug
```

Comment:
```suggestion
  resources :posts, only: %i[index create show], param: :slug
```

Reason: https://jaimy-simon.neetorecord.com/watch/fccdbd33bd79f6bdcca6

Not sure if i had explained the same to you.

Reply:
@jaimysimon Understood.